app/templates/admin/sales.hbs

```diff
@@ -1 +1,32 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="three wide column">
+      {{#tabbed-navigation isVertical=true}}
+        {{#link-to 'admin.sales.index'  class="item"}}
+          {{t 'By Events'}}
+        {{/link-to}}
+        {{#link-to 'admin.sales.organizers' class="item"}}
+          {{t 'By Organizer'}}
+        {{/link-to}}
+        {{#link-to 'admin.sales.locations' class="item"}}
+          {{t 'By Location'}}
+        {{/link-to}}
+        {{#link-to 'admin.sales.marketer' class="item"}}
```

Comment:
Single quotes for handlebar helpers please ... 
Like I said multiple times, please see https://github.com/fossasia/open-event-frontend/blob/development/docs/style-guide.md and follow the same ...